src/components/App.js

```diff
@@ -0,0 +1,31 @@
+import { loadFromLocal, saveToLocal } from '../lib/localStorage'
```

Comment:
I think the App should not be part of the components folder but rather directly in the src folder (as the pivot point for everything)

Reply:
Good point.